site/site_slim.go

```diff
@@ -4,12 +4,19 @@
 package site
 
 import (
-	"embed"
 	"io/fs"
+	"testing/fstest"
+	"time"
 )
 
-var slim embed.FS
-
 func FS() fs.FS {
-	return slim
+	// This is required to contain an index.html file for unit tests.
+	// Our unit tests frequently just hit `/` and expect to get a 200.
+	// So a valid index.html file should be expected to be served.
+	return fstest.MapFS{
+		"index.html": &fstest.MapFile{
+			Data:    []byte("Slim build of Coder, does not contain the frontend static files."),
+			ModTime: time.Now(),
+		},
+	}
 }
```

Comment:
@kylecarbs I had to add this so unit tests work. When they query `/` it was a 404 since `/` was a directory, which now is not allowed.